tests/integration/team_test.py

```diff
@@ -17,34 +17,88 @@
 # along with this program. If not, see <https://www.gnu.org/licenses/>.
 #
 
+from contextlib import contextmanager
+
 import pytest
 
 import libnmstate
-from libnmstate.error import NmstateNotImplementedError
+from libnmstate.error import NmstateLibnmError
 from libnmstate.schema import Interface
+from libnmstate.schema import InterfaceState
 from libnmstate.schema import InterfaceType
 from libnmstate.schema import Team
 
+from .testlib import assertlib
+from .testlib import cmd as libcmd
+
+
+DNF_REMOVE_NM_TEAM_CMD = ("dnf", "remove", "-y", "-q", "NetworkManager-team")
+DNF_INSTALL_NM_TEAM_CMD = ("dnf", "install", "-y", "-q", "NetworkManager-team")
+SYSTEMCTL_RESTART_NM_CMD = ("systemctl", "restart", "NetworkManager")
 
 TEAM0 = "team0"
 
-TEAM0_STATE = {
-    Interface.KEY: [
-        {
-            Interface.NAME: TEAM0,
-            Interface.TYPE: InterfaceType.TEAM,
-            Team.CONFIG_SUBTREE: {
-                Team.PORT_SUBTREE: [{Team.Port.NAME: "eth1"}]
+
+@pytest.mark.xfail(raises=NmstateLibnmError, reason="Team kmod not installed")
```

Comment:
This is not xfail, you should ask use to modprobe team.

And also do modprobe in `automation/run-tests.sh` like `function modprobe_ovs` did.

Reply:
There is no team kernel module for the Travis Ubuntu kernel, so it cannot be loaded there.

Reply:
As it is not possible to do have team kernel module for the Travis Ubuntu kernel, I am going to add a xskip with a condition on $CI. Does that work for you? Thanks

Reply:
Done!